assets/sass/protocol/base/elements/_links.scss

```diff
@@ -4,61 +4,51 @@
 
 @use '../../includes/lib' as *;
 
-:link {
-    color: $link-color;
+// Rely on source order for the stateful style overrides
+// 1. Visited
+// 2. Hover
+// 3. Active
+// 4. Focus
+
+
+a {
+    color: var(--link-color);
     text-decoration: underline;
 
-    &:hover,
-    &:focus,
-    &:active {
-        color: $link-color-hover;
+    &:visited {
+        color: var(--link-color-visited);
+    }
+
+    &:hover, &:active, &:focus {
+        /* stylelint-disable-next-line color-function-notation */
+        color: var(--link-color-with-state, hsl(from var(--link-color) h s calc(l - 10)));
```

Comment:
Demo of functional hover color approach:

1. We have a default that automatically darkens the lightness of the color by 10% 
2. We can override this default with another function (i.e. dark theme automatically lightens color) or another set value (i.e. white theme uses white)

Reply:
We could also do this with sass `color.scale` for better support but we have to use a sass variable as the input argument (CSS custom properties are not available as values at build time)
<img width="661" alt="need-sass-variable" src="https://github.com/user-attachments/assets/6e70db3a-520e-4d57-a2ed-a8487d249482" />

Reply:
This would allow us to drop support for `hover` tokens and (hopefully) simplify maintenance